pyspecdata/fourier/convolve.py

```diff
@@ -32,7 +32,7 @@ def convolve(self,axisname,filterwidth,convfunc = (lambda x,y: exp(-(x**2)/(2.0*
     else:
         self.ft(axisname)
     x = self.fromaxis(axisname)
-    myfilter = convfunc(x,filterwidth)
+    myfilter = convfunc(x,1/filterwidth)
     newdata = self*myfilter
```

Comment:
this is explained in the commit message -- the filter width is given in the CURRENT domain

Reply:
(That's why it's inverted)

Reply:
If I invert the filter width I get this, which does not show a difference in the convolution ![210730_diagnostic1](https://user-images.githubusercontent.com/31140790/127702757-fa4bafe0-7a39-4f9b-bba3-6a817ffe2e92.png)

Reply:
whereas using just filterwidth, I get this, which shows a difference
![210730_diagnostic2](https://user-images.githubusercontent.com/31140790/127703209-3ab38b12-f80d-44a6-8bee-2317390a33a0.png)

Reply:
wait I got mixed up - I think I understand this now... I will post back soon

Reply:
ok I have got it now. disregard above.

Reply:
I end up doing what I suggest on slack -- please check out and see that it works OK for you.